CHANGES.rst

```diff
@@ -1,7 +1,12 @@
 1.14.1 (unreleased)
 ===================
 
-- 
+resample
+--------
+
+- Apply a median filter to IVM weight array to better handle saturated
+  pixels. [#7563]
+
```

Comment:
```suggestion
```

now that #8671 is merged (switching change log handling to `towncrier`) this change log entry should be a file in `changes/` instead:
```shell
echo "Apply a median filter to IVM weight array to better handle saturated pixels." > changes/7563.resample.rst
```

(new PRs will include instructions on how to do this)

Reply:
(ignore this if this PR is no longer active or needed)